content/SCALE/SCALETutorials/Shares/SMB/ManageSMBShares.md

```diff
@@ -9,13 +9,17 @@ tags:
 - scaleacls
 ---
 
-
 {{< toc >}}
 
-
 To access SMB share management options from the **Sharing > Windows (SMB) Shares** screen you need to access the **Sharing >SMB** screen that lists all SMB shares on the system. 
 To access this, after going to **Shares**, click the **Windows (SMB) Shares <span class="material-icons">launch</span>** launch icon.
 
+{{< expand "What if my system is clustered" "v" >}}
+![ClusteredSMBShare](/images/SCALE/22.12/ClusteredSMBShare.png "Clustered SMB Share")
+
+If the system is part of a [cluster]({{< relref "SMBClustering.md" >}}), you cannot add or edit SMB shares. A tooltip in the **SMB Shares** widget states the share is configured through TrueCommand.
```

Comment:
I'd change this to state they configure the cluster share in TrueCommand otherwise it is more like UI ref content. Maybe something like this
```suggestion
If the system is part of a [cluster]({{< relref "SMBClustering.md" >}}), you cannot add or edit SMB shares. Configure the share through TrueCommand.
```